dist/api_data.py

```diff
@@ -491,6 +491,12 @@ def __ge__(self, other):
     Config('debug_mode', '', r'''
         control the settings of various extended debugging features''',
         type='category', subconfig=[
+        Config('aggressive_lookaside', 'false', r'''
+            if true, modify internal algorithms to change skew to force
+            lookaside eviction to happen more aggressively. This includes but
+            is not limited to not skewing newest, not favoring leaf pages,
+            and modifying the eviction score mechanism.''',
+            type='boolean'),
```

Comment:
Suggest just calling this "eviction", i.e. `debug_mode=(eviction)`, which better matches the internal name.

Reply:
Done.